### PR TITLE
Add helper script for updating ink! versions

### PR DESCRIPTION
**Pull request by HCastano**
I've had a little script like this in other projects just to make it dead easy when
updating versions (like in #864). Figured it would be worth proposing.

### CODE REVIEW

**Comment by cmichi** [comment]
Thanks for the PR! It's a good idea, but I would rather leave it out for the following reason:
the script will become obsolete as soon as we've reached a non-release-candidate state ‒ then we can upgrade individual crates on their own, i.e. they will have individual versions, making a global search/replace obsolete.

Fine with you?

**Comment by HCastano** [comment]
> the script will become obsolete as soon as we've reached a non-release-candidate state ‒ then we can upgrade individual crates on their own, i.e. they will have individual versions, making a global search/replace obsolete.

Well, it depends on how long that non-`rc` state is going to take to achieve. What do you have in mind?

**Comment by cmichi** [comment]
I'll close the PR, since we should do everything for getting from rc to stable.

**Comment by HCastano** [comment]
@cmichi I think this is relevant again 😉

File: scripts/bump-release-version.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+# One-liner to update between ink! release versions.
+#
+# Usage: ./bump-release-version.sh 3.0.0-rc3 3.0.0-rc4
+
+set -xeu
+
+OLD_VERSION=$1
+NEW_VERSION=$2
+
+find . -type f -name 'Cargo.toml' -exec sed -i -e "s/$OLD_VERSION/$NEW_VERSION/g" {} \;
```
Comment (cmichi) [inline scripts/bump-release-version.sh:12]: What about local `Cargo.lock`'s ?

**Comment by HCastano** [comment]
Gonna close this again 😅 

If we go with something like `cargo-release` or a hand rolled helper (like in #1211) we shouldn't need this

**Pull request closed without merging**